.github/actions/gcloud/index.js

```diff
@@ -0,0 +1,81 @@
+const core = require('@actions/core');
```

Comment:
As this appears to just be all `exec`. Should we just do it as a bash action instead? The added indirection feels weird.

Reply:
I agree bash would be the right language here. But custom actions can only be built in javascript, which run directly on the Github VM, or a container that would run its own sandbox. The latter is geared more towards isolated tasks, whereas javascript actions modify the VM environment and can interact better with the other steps in the job.

I tried doing this with a bash script inside a container, that would set the gcloud, kubectl and docker configs in a directory from the VM that is mounted in the container, and then in the other job steps reconfigure those tools to point to those non-standard config locations, but it was messy and let to lots of repetition.

Reply:
Good call, I got the wrappers all confused.